qiskit_code_assistant_jupyterlab/handlers.py

```diff
@@ -94,98 +121,166 @@ def post(self):
 class ModelsHandler(APIHandler):
     @tornado.web.authenticated
     def get(self):
-        url = url_path_join(runtime_configs["service_url"], "models")
-
-        try:
-            r = requests.get(url, headers=get_header())
-            r.raise_for_status()
-        except requests.exceptions.HTTPError as err:
-            self.set_status(err.response.status_code)
-            self.finish(json.dumps(err.response.json()))
+        if runtime_configs["is_openai"]:
+            url = url_path_join(runtime_configs["service_url"], "v1", "models")
```

Comment:
not a big deal but should `v1` be a variable instead? that way if for some reason the version changes or differs in future implementations it would be easier to update the variable.

Reply:
ah, nice catch, spaced on the fact that v1 indicated version 1 of the API